src/css/components/activity_log.scss

```diff
@@ -7,21 +12,38 @@
 
 .activity_log-item {
   align-items: center;
-  background-size: 2em;
-  background-repeat: no-repeat;
-  background-position-y: 40%;
   border-bottom: 1px solid #ccc;
   display: flex;
   flex-wrap: wrap;
   margin: 0;
-  padding-left: 2em;
+  padding-left: 1em;
```

Comment:
Why are these `em` instead of `rem`? There's not font size set here, is there one higher up?